ramls/pick-slips-response.json

```diff
@@ -0,0 +1,89 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Collection of items",
+  "description": "Collection of items",
+  "type": "object",
+  "properties": {
+    "items": {
+      "description": "Collection of items",
+      "id": "items",
+      "type": "array",
+      "items": {
+        "title": "An item",
+        "description": "Information about an item",
+        "type": "object",
+        "properties": {
+          "id": {
+            "description": "ID of the item",
+            "type": "string",
+            "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$"
```

Comment:
Please can this be extracted to a [general uuid pattern](https://github.com/folio-org/mod-inventory-storage/blob/master/ramls/uuid.json) as has been done in other modules

Reply:
done